yt_dlp/extractor/rtp.py

```diff
@@ -29,6 +29,17 @@ class RTPIE(InfoExtractor):
         \s*\.\s*join\(\s*(?:""|'')\s*\)\s*\)\s*\)
     ''')
 
+    __HEADERS = {
```

Comment:
I disagree with the use of `__` here, but don't really care enough to argue over it's "correct usage"